src/PYPPhoneticEditor.cc

```diff
@@ -647,3 +661,29 @@ PhoneticEditor::moveCursorRightByWord (void)
     update ();
     return TRUE;
 }
+
+void
+PhoneticEditor::updateFullPinyinBuffer (void)
```

Comment:
Is it possible to move this method into class CloudCandidates?

And use pinyin_get_full_pinyin_auxiliary_text function directly?